### ISSUE TITLE
SVT: Error starting Codewind on VSCode on Ubuntu 18.04

### ISSUE THREAD

**Issue by beccabau** [eclipse/codewind#730]
**Codewind version:** 0.4.0 
**OS:** Ubuntu 18.04 (Linux x64 4.15.0-65-generic)

**Che version:**  n/a
**IDE extension version:** 0.4.0
**IDE version:** Visual Studio Code Version 1.38.1
**Kubernetes cluster:** n/a

**Description:**
When installing Codewind extension from the Vs Code Marketplace, I get this panic: runtime error message ->
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x842d2f]

goroutine 1 [running]:
github.com/eclipse/codewind-installer/actions.StatusCommand(0xc00009cb00)
	/home/jenkins/src/github.com/eclipse/codewind-installer/actions/status.go:29 +0x2f
github.com/eclipse/codewind-installer/actions.Commands.func4(0xc00009cb00, 0x0, 0xc00020d410)
	/home/jenkins/src/github.com/eclipse/codewind-installer/actions/commands.go:116 +0x2b
github.com/eclipse/codewind-installer/vendor/github.com/urfave/cli.HandleAction(0x8b6500, 0x9a6538, 0xc00009cb00, 0xc00009cb00, 0x0)
	/home/jenkins/src/github.com/eclipse/codewind-installer/vendor/github.com/urfave/cli/app.go:514 +0xbe
github.com/eclipse/codewind-installer/vendor/github.com/urfave/cli.Command.Run(0x987d23, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0x999ca8, 0x29, 0x0, ...)
	/home/jenkins/src/github.com/eclipse/codewind-installer/vendor/github.com/urfave/cli/command.go:171 +0x4a4
github.com/eclipse/codewind-installer/vendor/github.com/urfave/cli.(*App).Run(0xc0002dc1c0, 0xc00001c090, 0x3, 0x3, 0x0, 0x0)
	/home/jenkins/src/github.com/eclipse/codewind-installer/vendor/github.com/urfave/cli/app.go:265 +0x70b
github.com/eclipse/codewind-installer/actions.Commands()
	/home/jenkins/src/github.com/eclipse/codewind-installer/actions/commands.go:313 +0x1796
main.main()
	/home/jenkins/src/github.com/eclipse/codewind-installer/main.go:18 +0x25
```

*I believe that the codewind-workspace is not being created on install.* I checked under root/ and did not see any code-workspace directories (which I expected to see).

**Steps to reproduce:**
1. Uninstall any previous codewind installs and `docker system prune -a` to start "clean"
2. Go to VS Code marketplace and install Codewind, follow any prompts and instructions as part of the install process
3. Wait
4. When codewind tries to start, it should flash a panic: runtime error

![image](https://user-images.githubusercontent.com/13971834/67037515-a9e74e80-f0eb-11e9-94a5-d15f14e6750c.png)

Jag is planning to check on his linux machine to see if he can recreate.

**Workaround:** 

no workaround yet, just documenting in case this is a bigger issue.

**Comment by beccabau** [comment]
I also can't start codewind with 0.5.0, but not sure if it's due to the same problem or not (I think maybe because the `codewind-workspace` directory is not being created). I could open a separate issue if y'all think it's different.
![image](https://user-images.githubusercontent.com/13971834/67037620-dac78380-f0eb-11e9-981a-0b7b0de63164.png)

**Comment by beccabau** [comment]
This is the full suite of informational messages I get: 
![image](https://user-images.githubusercontent.com/13971834/67039040-938ec200-f0ee-11e9-8abf-b02fdf586841.png)

**Comment by tetchel** [comment]
this is https://github.com/eclipse/codewind/issues/625#issuecomment-538547910

the workaround was to delete `.config/deployments.json` 
but i'm not sure how you are hitting this, it should not be possible unless you did some testing during that few days where the cwctl was broken

**Comment by jopit** [comment]
The original description talks about installing the 0.4.0 vscode extension.  I'm wondering if the `.config/deployments` file was created by 0.5.0, and 0.4.0 can't handle it.

**Comment by tetchel** [comment]
well, that's not a problem since no user is going from 0.4 -> 0.5

**Comment by tetchel** [comment]
and yes, the codewind-workspace will not get created since you hit the nil pointer bug first. so, I think we can close this as invalid

**Comment by tetchel** [comment]
in any case the nil pointer is fixed in 0.5 so i think we can close this
